parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -1017,6 +987,52 @@ pub mod pallet {
 				},
 			}
 		}
+
+		pub fn participant_pubkeys(
+			sync_committee_bits: &[u8],
+			sync_committee_pubkeys: &Vec<PublicKeyPrepared>,
+		) -> Result<Vec<PublicKeyPrepared>, Error<T>> {
```

Comment:
We know that the length of `sync_committee_bits` and `sync_committee_pubkeys` is equal. So we don't need to to return any possibility of errors.

```suggestion
		) -> Vec<PublicKeyPrepared> {
```